File: api-food-app/src/main/java/com/apifoodapp/Model/Comida.java
package com.apifoodapp.Model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Data;

@Entity
@Table(name = "comida")
@Data
public class Comida {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String nome;
    private Double preco;
    private Boolean favorito;
    private String imageUrl;
    private String origem;
    private String tempoDePreparo;

}

File: projeto-food-app/src/app/shared/models/Comida.ts
export class Comida{
  id!: number;
  nome!: string;
  preco!: number;
  tags?: string[];
  favorito: boolean = false;
  estrelas: number = 0;
  imageUrl!: string;
  origem!: string;
  tempoDePreparo!: string;
}
